integration/fabric/iou/iou_test.go

```diff
@@ -34,6 +36,8 @@ var _ = Describe("EndToEnd", func() {
 			Expect(err).NotTo(HaveOccurred())
 			// Start the integration ii
 			ii.Start()
+			// Sleep for a while to allow the networks to be ready
+			time.Sleep(20 * time.Second)
```

Comment:
oh this seems to be a dirty trick to tame the flacky IOU :)

Reply:
I don't see another way to learn when libp2p is in sync :(